src/index.js

```diff
@@ -0,0 +1,25 @@
+document.getElementById('addItem').addEventListener('click', () => addItem());
```

Comment:
It's not worrying, and it even depends on each person's taste, but in this case you are passing an arrow function as a callback that calls a function, you can simply pass the function as a callback:

```suggestion
document.getElementById('addItem').addEventListener('click', addItem);
```

Reply:
yea that makes more sense